resolwe_bio/processes/support_processors/multiqc.py

```diff
@@ -0,0 +1,242 @@
+            """Options."""
+
+            dirs = BooleanField(
+                label='--dirs',
+                default=True,
+                description='Prepend directory to sample names.',
+            )
+
+            dirs_depth = IntegerField(
+                label='--dirs-depth',
+                default=-1,
+                description='Prepend a specified number of directories to sample names. Enter a '
+                            'negative number to take from start of path.',
+            )
+
+            fullnames = BooleanField(
+                label='--fullnames',
+                default=False,
+                description='Do not clean the sample names (leave as full file name).',
+            )
+
+            config = BooleanField(
+                label='Use configuration file',
+                default=True,
+                description='Use Genialis configuration file for MultiQC report.',
+            )
+
+            cl_config = StringField(
+                label='--cl-config',
+                required=False,
+                description='Enter text with command-line configuration options to override the '
+                            'defaults (e.g. custom_logo_url: https://www.genialis.com).',
+            )
+
+        advanced = GroupField(Advanced, label="Advanced options")
+
+    class Output:
+        """Output fields."""
+
+        report = FileHtmlField(label='MultiQC report')
+        report_data = DirField(label='Report data')
+
+    def run(self, inputs, outputs):
+        """Run the analysis."""
+        samples = []
+        species = []
+        build = []
+        unsupported_data = []
+
+        for d in inputs.data:
+            sample_dir = d.entity_name
+
+            if sample_dir and not os.path.isdir(sample_dir):
+                os.mkdir(sample_dir)
+
+            try:
+                if d.entity_name and d.species and d.build:
+                    samples.append(d.entity_name)
+                    species.append(d.species)
+                    build.append(d.build)
+            except AttributeError:
+                pass
+
+            if d.type.startswith('data:reads:fastq:single'):
+                for fq_report in d.fastqc_archive:
+                    name = os.path.basename(fq_report.path)
+                    create_symlink(fq_report.path, os.path.join(sample_dir, name))
+
+            elif d.type.startswith('data:reads:fastq:paired'):
+                for fq_report in d.fastqc_archive + d.fastqc_archive2:
+                    name = os.path.basename(fq_report.path)
+                    create_symlink(fq_report.path, os.path.join(sample_dir, name))
+
+            elif d.type == 'data:alignment:bam:star:':
+                stats_file = os.path.basename(d.stats.path)
+                assert stats_file.endswith('_stats.txt')
+                bam_name = stats_file[:-10]
+
+                if d.build == 'rRNA':
+                    rrna_report = f'{bam_name}.rRNA.Log.final.out'
```

Comment:
Would underscores make filename any more clear?

```
f'{bam_name}.rRNA.Log.final.out'
f'{bam_name}_rRNA_Log_final.out'
```

Also consider if you want to introduce capital letters to the filename (i.e. `Log`). It makes sense to capitalize RNA, but I'm not sure about Log.
Also, is final relevant here? Are there non-final reports being outputted?

Reply:
MultiQC recognises the report files from various tools based on either content, or based on output file names. In this case, the `Log.final.out` is required for MultiQC to recognise and parse the input.